packages/offix-client/src/OfflineClient.ts

```diff
@@ -135,13 +134,13 @@ export class OfflineClient implements ListenerProvider {
         variables: mutationOptions.variables
       })
 
+      // TODO we could probably move the conflictBase directly on to the mutationOptions.
+      // does createMutationOptions living in offix-cache make sense then though?
```

Comment:
Generally, the side effects of the refactor to the scheduler are that people will not need to bother with Apollo cache. Apollo proposes to use graphql (and InMemoryCache) for a persisting state. I think people want to simply have state available (and persisted) - what I mean for us is that optimistic responses can become just optional feature that people can add (along with update methods) not the element on the critical path. 
Other offline providers use the same approach

Reply:
TL;DR conflict base in mutationOptions will be against responsibility. Our base will be taken once someone will schedule offline operation